content/2023/contents/home.mdx

```diff
@@ -1,5 +1,9 @@
 import Image from 'next/image';
+import partner_numfocus from '@/assets/NumFocus.png';
+import partner_pydata from '@/assets/PyData.png';
+import partner_humbledata from '@/assets/humble-data.png';
 import squareImg from '@/assets/sw23_square.jpg';
+import sponsor_xvideo from '@/assets/xvideo.png';
```

Comment:
please naming variables via camelCase

Reply:
however, I'm going to merge this PR first